modules/contentbox/models/modules/ModuleService.cfc

```diff
@@ -378,24 +378,24 @@ component extends="cborm.models.VirtualEntityService" accessors="true" singleton
 		};
 
 		// Register Core
-		qCoreModules.each( function( row ){
+		for(row in qCoreModules){
```

Comment:
Any reason why this change?

Reply:
Luis ... we have done this because of Each loop on query object is not supported in CF11

Pleae see this error we get.

![cbissue](https://user-images.githubusercontent.com/12493160/37138574-93deba6e-22d0-11e8-8160-2eb273dd3599.png)

Reply:
Arggghhh good old Adobe